.github/workflows/examples.yml

```diff
@@ -181,4 +181,4 @@ jobs:
         if: ${{ matrix.dir == 'deno-deploy' }}
         run: |
           cd ./examples/${{ matrix.dir }}
-          pnpm start-server-and-test "deno run --allow-net=:8000 --allow-env ./src/index.ts" http://127.0.0.1:8000 "deno run --allow-net ./src/client.ts"
\ No newline at end of file
+          pnpx start-server-and-test "deno run --allow-net=:8000 --allow-env ./src/index.ts" http://127.0.0.1:8000 "deno run --allow-net ./src/client.ts"
```

Comment:
I think we should use `pnpm exec` instead 
`pnpx` has a deprecated notice on the pnpm website
https://pnpm.io/6.x/pnpx-cli

Reply:
If that works, sure, but we don't wanna add a `package.json` to this.

Maybe deno has an equivalent package?

Reply:
From the `pnpm exec` docs:
> Execute a shell command in scope of a project.

So that won't work. I'm trying with `pnpm dlx` instead which should work properly

Reply:
Working fine with `pnpm dlx`, thanks for the suggestion :)